### PR TITLE
[ES] Add `Performance troubleshooting`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Performance_troubleshooting/es.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Solución para los problemas de rendimiento de osu!
+
+osu! hace todo lo que está en su poder para funcionar con una configuración predeterminada adecuada y te avisa de forma proactiva si estás haciendo/ejecutando algo que podría hacer que el rendimiento de tu juego disminuyera.
+
+Lamentablemente, debido a la variedad de hardware que usa osu!, es muy difícil garantizar un rendimiento óptimo para todos los usuarios. Esta página pretende ayudarte a autodiagnosticar los problemas de rendimiento que tengas, pero también proporciona términos específicos que puedes aprender (y comprobaciones que puedes realizar) para ayudarnos a ayudarte cuando solicites ayuda.
```
Comment (agatemosu) [inline wiki/Performance_troubleshooting/es.md:5]:
```suggestion
Lamentablemente, debido a la variedad de hardware que ejecuta osu!, es muy difícil garantizar un rendimiento óptimo para todos los usuarios. Esta página pretende ayudarte a autodiagnosticar los problemas de rendimiento que tengas, pero también proporciona términos específicos que puedes aprender (y comprobaciones que puedes realizar) para ayudarnos a ayudarte cuando solicites ayuda.
```

File: wiki/Performance_troubleshooting/es.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Solución para los problemas de rendimiento de osu!
+
+osu! hace todo lo que está en su poder para funcionar con una configuración predeterminada adecuada y te avisa de forma proactiva si estás haciendo/ejecutando algo que podría hacer que el rendimiento de tu juego disminuyera.
```
Comment (agatemosu) [inline wiki/Performance_troubleshooting/es.md:3]:
```suggestion
osu! hace todo lo puede para funcionar con una configuración predeterminada adecuada y te avisa de forma proactiva si estás haciendo/ejecutando algo que podría hacer que el rendimiento de tu juego disminuyera.
```

File: wiki/Performance_troubleshooting/es.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+- Abrir el administrador de tareas y ordenar la lista (en Detalles) por CPU. Busca cualquier uso elevado de la CPU durante los periodos de baja velocidad de fotogramas, por ejemplo, por parte de un escáner de virus o malware en segundo plano. Puedes ejecutar osu! en modo ventana o dejar el administrador de tareas en una segunda pantalla para diagnosticar esto mientras juegas o reproduces un beatmap en modo de juego automático.
+
+## Limitadores de fotogramas
+
+osu! ofrece múltiples opciones para los limitadores de fotogramas. Cada uno de ellos tiene sus ventajas y desventajas. Ten en cuenta que, independientemente de la opción elegida, osu! limitará la velocidad de fotogramas en los menús para conservar el uso de la CPU.
+
+### VSync
+
+Este limitador de fotogramas se basa en un mecanismo de sincronización vertical proporcionado por el controlador. Garantiza que no se produzca ninguna imagen rasgada, pero añade 1-2 fotogramas de latencia debido a la necesidad de almacenar los fotogramas en el búfer antes de mostrarlos.
```
Comment (agatemosu) [inline wiki/Performance_troubleshooting/es.md:70]:
```suggestion
Este limitador de fotogramas se basa en un mecanismo de sincronización vertical proporcionado por el controlador. Garantiza que no se produzca ninguna imagen rasgada, pero añade 1 o 2 fotogramas de latencia debido a la necesidad de almacenar los fotogramas en el búfer antes de mostrarlos.
```

File: wiki/Performance_troubleshooting/es.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+Este es un limitador de fotogramas que no hace nada para limitar la velocidad de fotogramas. osu! renderizará tan rápido como pueda tu CPU/GPU.
+
+Si bien puede parecer la mejor opción (y de hecho puede serlo, si tu hardware puede soportarlo), hay algunas cosas que debes tener en cuenta:
+
+- Al renderizar más fotogramas, hay un mayor riesgo de sobrecarga del [recolector de basura](https://es.wikipedia.org/wiki/Recolector_de_basura).
+- Si llevas tu CPU y GPU al límite, existe la posibilidad de que se ralenticen, lo que reducirá tu rendimiento general.
+- Por lo general, las GPU no están diseñadas para renderizar más de 1000 fotogramas por segundo, por lo que pueden producirse comportamientos inesperados, como un menor rendimiento o una latencia más alta de lo que se esperaría.
+
+Aun así, si la generación de calor y el posible sobreesfuerzo de tu hardware no se consideran un problema, `Ilimitado` puede ser la opción que te proporcione los fotogramas más estables.
```
Comment (agatemosu) [inline wiki/Performance_troubleshooting/es.md:96]: maaaaal

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**